api/main.py

```diff
@@ -30,7 +31,10 @@ def setup_logging():
 
 logger = logging.getLogger(__name__)
 
-app = FastAPI(swagger_ui_parameters={"tryItOutEnabled": True})
+app = FastAPI(
+    swagger_ui_parameters={"tryItOutEnabled": True},
+    root_path=os.getenv("FASTAPI_ROOT_PATH", ""),
```

Comment:
If I set a root path of `/123`, I now get responses at both:
```
http://localhost:8008/123/collections
http://localhost:8008/collections
```

Looking at the [docs](https://fastapi.tiangolo.com/advanced/behind-a-proxy/), I am not sure this is what `root_path` is intended for (and probably explains the strange behavior).

Reply:
I think is is expected behavior as explained by the docs. When this is set, you are expected to route through the proxy.